test/test_functions

```diff
@@ -28,6 +28,13 @@ CVMFS_TEST_S3_CONFIG=${CVMFS_TEST_S3_CONFIG:=}
 CVMFS_TEST_STRATUM0=${CVMFS_TEST_STRATUM0:=}
 CVMFS_TEST_SERVER_CACHE=${CVMFS_TEST_SERVER_CACHE:=}
 
+# This OPT variables are for the benchmark options under the benchmark_tests folder
+CVMFS_OPT_WARM_CACHE=${CVMFS_OPT_WARM_CACHE:=yes}
+CVMFS_OPT_ITERATIONS=${CVMFS_OPT_ITERATIONS:=1}
+CVMFS_OPT_TEST_TYPE=${CVMFS_OPT_TEST_TYPE:=callgrind} # callgrind / memcheck
+CVMFS_OPT_CONFIG_FILE=${CVMFS_OPT_CONFIG_FILE:=}
+CVMFS_OPT_OUTPUT_DIR=${CVMFS_OPT_OUTPUT_DIR:=~/cvmfs_benchmarks}
```

Comment:
In run.sh, line 178 (sudo rm -rf "$workdir" >> $logfile) it removes everything inside the workspace. That's the reason I wanted a place that I know it won't be deleted.